zap/src/output/luau/server.luau

```diff
@@ -1,22 +1,4 @@
-local Players = game:GetService("Players")
 
-if RunService:IsClient() then
-	error("Cannot use the server module on the client!")
-end
-
-local reliable = ReplicatedStorage:FindFirstChild("ZAP_RELIABLE")
-if reliable == nil then
-	reliable = Instance.new("RemoteEvent")
-	reliable.Name = "ZAP_RELIABLE"
-	reliable.Parent = ReplicatedStorage
-end
-
-local unreliable = ReplicatedStorage:FindFirstChild("ZAP_UNRELIABLE")
-if unreliable == nil then
-	unreliable = Instance.new("UnreliableRemoteEvent")
-	unreliable.Name = "ZAP_UNRELIABLE"
-	unreliable.Parent = ReplicatedStorage
-end
 
 local player_map = {}
```

Comment:
Let's completely get rid of `server.luau` if you're getting rid of `client.luau`.

Reply:
Sounds good to me